### ISSUE TITLE
run bash tools/e2e_smoke_test.sh  

### ISSUE THREAD

**Issue by queyibuke** [google-research/circuit_training#66]
'''docker run --rm -v ${REPO_ROOT}:/workspace --workdir /workspace circuit_training:core     bash tools/e2e_smoke_test.sh --root_dir /workspace/logs''' 
I run  test py using above command,I am getting this error 

I1021 00:58:03.773555 139922392946496 NonLinearPlace.py:609] optimizer nesterov takes 1.601 seconds
I1021 00:58:03.773948 139922392946496 dreamplace_core.py:50] Last Dreamplace metric: iteration  414, ( 414,  0,  0), Obj 6.806722E+05, DensityWeight 1.744282E+00, HPWL 7.100381E+05, Overflow 9.989433E-02, MaxDensity 7.041E+00, gamma 2.227645E+00, time 0.464ms
W1021 00:58:04.419603 139922392946496 plc_client.py:99] JSONDecode Error for GetMacroAndClusteredPortAdjacency 
 Expecting ',' delimiter: line 1 column 255809 (char 255808)
I1021 00:58:04.419809 139922392946496 plc_client.py:101] Looking for more data for GetMacroAndClusteredPortAdjacency on connection:0/256
W1021 00:58:04.436443 139922392946496 plc_client.py:99] JSONDecode Error for GetMacroAndClusteredPortAdjacency 
 Expecting ',' delimiter: line 1 column 475073 (char 475072)
I1021 00:58:04.436555 139922392946496 plc_client.py:101] Looking for more data for GetMacroAndClusteredPortAdjacency on connection:1/256
W1021 00:58:04.459675 139922392946496 plc_client.py:99] JSONDecode Error for GetMacroAndClusteredPortAdjacency 
 Expecting value: line 1 column 694337 (char 694336)
I1021 00:58:04.459773 139922392946496 plc_client.py:101] Looking for more data for GetMacroAndClusteredPortAdjacency on connection:2/256
W1021 00:58:04.484143 139922392946496 plc_client.py:99] JSONDecode Error for GetMacroAndClusteredPortAdjacency 
 Expecting ',' delimiter: line 1 column 913601 (char 913600)
I1021 00:58:04.484235 139922392946496 plc_client.py:101] Looking for more data for GetMacroAndClusteredPortAdjacency on connection:3/256
W1021 00:58:04.516166 139922392946496 plc_client.py:99] JSONDecode Error for GetMacroAndClusteredPortAdjacency 
 Expecting ',' delimiter: line 1 column 1132865 (char 1132864)
I1021 00:58:04.516258 139922392946496 plc_client.py:101] Looking for more data for GetMacroAndClusteredPortAdjacency on connection:4/256
W1021 00:58:04.552229 139922392946496 plc_client.py:99] JSONDecode Error for GetMacroAndClusteredPortAdjacency 
 Expecting value: line 1 column 1352129 (char 1352128)
I1021 00:58:04.552327 139922392946496 plc_client.py:101] Looking for more data for GetMacroAndClusteredPortAdjacency on connection:5/256
W1021 00:58:04.594193 139922392946496 plc_client.py:99] JSONDecode Error for GetMacroAndClusteredPortAdjacency 
 Expecting value: line 1 column 1571393 (char 1571392)
I1021 00:58:04.594290 139922392946496 plc_client.py:101] Looking for more data for GetMacroAndClusteredPortAdjacency on connection:6/256
W1021 00:58:04.645808 139922392946496 plc_client.py:99] JSONDecode Error for GetMacroAndClusteredPortAdjacency 
 Expecting ',' delimiter: line 1 column 1790657 (char 1790656)
I1021 00:58:04.645917 139922392946496 plc_client.py:101] Looking for more data for GetMacroAndClusteredPortAdjacency on connection:7/256
I1021 00:58:05.016601 139922392946496 observation_extractor.py:349] Pad a tensor with shape (8312,) by 42000
I1021 00:58:05.017001 139922392946496 observation_extractor.py:363] Pad a tensor with shape (953,) by 3500
W1021 00:58:05.944567 139922392946496 ppo_agent.py:332] Only tf.keras.optimizers.Optimiers are well supported, got a non-TF2 optimizer: None
I1021 00:58:12.566837 139922392946496 ppo_collect_lib.py:144] Collecting at model_id: 2
I1021 00:58:39.606090 139922392946496 environment.py:595] Actions took before becoming infeasible: [9167, 7247, 9652, 8249, 9406, 7243, 9030, 9413, 9421, 9911, 6478, 9934, 7746, 8270, 9152, 6836, 7475, 8769, 9263, 9668, 8010, 9677, 9915, 6462, 7215, 9671, 8259, 6578, 6846, 8773, 9008, 7855, 6731, 9908, 8395, 9289, 9528, 7107, 7097, 7877, 10052, 8759, 7484, 7353, 6339, 6217, 9403, 8765, 7218, 6832, 7730, 7472, 8112, 9018, 6206, 6326, 6862, 7608, 7625, 6595, 7992, 7094, 7363, 9904, 6982, 8783, 9035, 6726, 8652, 6320, 6581, 9140, 8135, 8505, 7758, 9149, 9793, 7488, 9272, 7871, 9802, 9519, 7500, 8624, 7868, 9015, 8002, 6839, 10049, 6585, 8390, 8124, 7494, 7100, 9397, 8014, 8373, 8762, 6330, 10057, 8510, 6986, 8776, 6475, 6222, 9410, 8756, 9155, 9661, 7240, 9546, 6842, 8513, 7104, 9394, 6850, 8117, 8370, 9918, 6472, 6214, 7733, 6323, 7350, 8527, 8367, 8127, 6575, 7503]
I1021 00:58:39.606333 139922392946496 environment.py:599] Using DREAMPlace mixed-size placer for the rest of the macros and std cell clusters.
I1021 00:58:40.591566 139922392946496 placedb_plc.py:56] Reinitialized the PlaceDB.

'''Could anyone suggest me how to resolve it'''
